typings/connectProps.d.ts

```diff
@@ -38,6 +38,7 @@ interface ConnectProps {
   userFeatures?: object
 }
 interface ClientConfigType {
+  _initialValues: string
```

Comment:
Nit: Should this be an object instead?

Reply:
Nope, a string is correct, because it is stored as a JSON string.  It was done this way to prevent object references causing problems.

When the selector is used it turns it back into an object.

Reply:
☝🏻 `JSON.stringify` is one of may ways to deep copy an object that is purely used for values.

Reply:
The only issue with this approach is that using `_initialValues` requires accessing it through a selector, since the value is stored as a string. Essentially, there are two different representations of `_initialValues` depending on how it’s accessed. I’m wondering if there’s a way we could deep copy the object, avoiding reference issues, So the value can be stored and used as an actual object instead.

Reply:
While it's not a convention that everyone follows, the `_` in front the the `_initialValues` is meant to mean it's private and should be directly used.  It's somewhat of a standard.

So in this case, the "right" way to use the initialValues would be to use the Selector.

Reply:
@mwclemy , if we run into issues in the future, lets revisit this.  I jus didn't want to have to create a deep copy function, or introduce a new library to do so.

This is a dependency free way to get what we need.

Reply:
Just wanted to throw that out there. Since this is an initial MR that lays the groundwork for future work, it’s definitely not a blocker. I’m totally fine addressing it in a future MR if needed.

I did notice that the native `structuredClone` method or Lodash’s `_.cloneDeep` can be used for deep copying if object reference issues come up.